candle-core/src/quantized/cuda.rs

```diff
@@ -2,21 +2,46 @@ use super::{GgmlDType, QStorage};
 use crate::{backend::BackendDevice, cuda_backend::WrapErr};
 use crate::{CudaDevice, CudaStorage, Result};
 
-use cudarc::driver::{CudaSlice, DeviceSlice};
+use cudarc::driver::{CudaSlice, CudaView, DeviceSlice};
 
 pub struct QCudaStorage {
     data: CudaSlice<u8>,
     dtype: GgmlDType,
     device: CudaDevice,
 }
 
+pub const FORCE_DMMV: bool = false;
```

Comment:
Could this maybe be an environment variable or ideally influenced by the compute cap? The `__dp4a` intrinsic is used only supported on CC>610. I'm not sure what the current minimum compute cap for Candle is, but it seems like it would be better to avoid increasing it?

Reply:
Looks like __nv_bfloat16 is CC>800, so this is not a problem.